etc/icinga2/conf.d/commands.conf

```diff
@@ -43,6 +43,9 @@ object NotificationCommand "mail-host-notification" {
       required = true
       value = "$notification_hoststate$"
     }
+    "-X" = {
+      required = false
+      value = "$notification_hostnotes$"
     "-t" = {
```

Comment:
A `}` is missing here. Also, you don't even have to put the `required = false` here for optional arguments, see the first few ones in this block for example.